plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/core/server/OpenShiftServerExtendedProperties.java

```diff
@@ -0,0 +1,32 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Red Hat Inc..
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Red Hat Incorporated - initial API and implementation
+ *******************************************************************************/
+package org.jboss.tools.openshift.core.server;
+
+import org.eclipse.core.runtime.IAdaptable;
+import org.jboss.ide.eclipse.as.core.server.internal.extendedproperties.ServerExtendedProperties;
+
+/**
+ * @author Andre Dietisheim
+ */
+public class OpenShiftServerExtendedProperties extends ServerExtendedProperties {
+
+	public OpenShiftServerExtendedProperties(IAdaptable adaptable) {
+		super(adaptable);
+	}
+	
+	public boolean hasWelcomePage() {
+		return true;
+	}
+	
+	public String getWelcomePageUrl() {
+		return null;
+	}
+}
```

Comment:
add the following so that the XML Configuration and Filesets noded don't show up:

	
	@Override
	public boolean allowConvenienceEnhancements() {
		return false;
	}

See https://issues.jboss.org/browse/JBIDE-21116